controllers/v1/static-links.js

```diff
@@ -0,0 +1,232 @@
+/**
+ * name : staticLinks.js
+ * author : Rakesh
+ * created-date : 28-Oct-2020
+ * Description : Static links related information.
+ */
+
+// Dependencies
+const csv = require("csvtojson");
+const staticLinksHelper = require(MODULES_BASE_PATH + "/static-links/helper")
+const FileStream = require(ROOT_PATH + "/generics/file-stream");
+
+/**
+    * StaticLinks
+    * @class
+*/
+module.exports = class StaticLinks extends Abstract {
+  constructor() {
+    super(schemas["static-links"]);
+  }
+
+  static get name() {
+    return "static-links";
+  }
+
+  /**
+  * @api {get} /kendra/api/v1/static-links/list Static Link list
+  * @apiVersion 1.0.0
+  * @apiName Static Link list
+  * @apiGroup Static Links
+  * @apiHeader {String} X-authenticated-user-token Authenticity token
+  * @apiSampleRequest /kendra/api/v1/static-links/list
+  * @apiParamExample {json} Response:
+  * {
+  *  "message": "Static Links fetched successfully.",
+    "status": 200,
+    "result": [
+      privacyPolicy": {
+            "_id": "5d259439a9bc1209d0184390",
+            "value": "privacyPolicy",
+            "link": "https://shikshalokam.org/wp-content/uploads/2019/01/data_privacy_policy.html",
+            "title": "Privacy Policy"
+        },
+        "termsOfUse": {
+            "_id": "5d259439a9bc1209d0184391",
+            "value": "termsOfUse",
+            "link": "https://shikshalokam.org/wp-content/uploads/2019/05/Final-ShikshaLokam-Terms-of-Use-MCM-08052019-Clean-copy-1.html",
+            "title": "Terms of Use"
+        }
+      
+    ]
+  }
+  * @apiUse successBody
+  * @apiUse errorBody
+  */
+
+  /**
+ * List static links.
+ * @method
+ * @name list
+ * @returns {Array} List of all static links. 
+ */
+
+  list(req) {
+    return new Promise(async (resolve, reject) => {
+
+      try {
+
+        let result = await staticLinksHelper.list(
+          req.headers.apptype,
+          req.headers.appname
+        )
+
+        return resolve(result);
+
+      } catch (error) {
+
+        return reject({
+          status: error.status || httpStatusCode.internal_server_error.status,
+          message: error.message || httpStatusCode.internal_server_error.message,
+          errorObject: error
+        });
+
+      }
+
+
+    })
+  }
+
+  /**
+ * @api {post} /kendra/api/v1/static-links/bulkCreate Upload Static Links Information CSV
```

Comment:
@rakeshSgr - Please do look at samiksha apis for the same once to compare if any changes required.

Cc - @aman-tunerlabs